index.html

```diff
@@ -109,7 +109,8 @@ <h2 class="header-text">Multi-Post Stories</h2>
             <a href="#" class="multiposts-btn main-btn">See Project</a>
           </div>
         </div>
-        <div class="projects">
+        <div id="project-cards-container"></div>
+        <!-- <div class="projects">
```

Comment:
Please remove the HTML codes for the projects section as the previous reviewer requested and don't leave any uncommented code in your codebase. So you can have a neat and professional codebase 👨‍💼